### PR TITLE
Update 08_i-don-t-want-everyone-to-know-where-i-live-what-do-i-do.md

### PR DESCRIPTION
**Pull request by sylee-**

### CODE REVIEW

**Review by Haribishop — APPROVED** [review]

**Review by emfet — APPROVED** [review]
👍

File: 02_profiles/08_i-don-t-want-everyone-to-know-where-i-live-what-do-i-do.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 We recommend you choose a location that is in your general area, but not specific to your current location.
 
-If you really want to be off the radar you can always just specify your country as "Antarctica". There are more kinksters than you would think living in Antarctica. Apparently the penguin population is getting very nervous... For the leather, you pervs, the leather!!!
+If you really want to be off the radar, you can always just specify your country as "Antarctica”. There are more kinksters than you would think of living in Antarctica.
```
Comment (DarklingMuse) [inline 02_profiles/08_i-don-t-want-everyone-to-know-where-i-live-what-do-i-do.md:5]: Grammar is a bit iffy on this last line here, I'd suggest re-phrasing it to: 'There are more kinksters living in Antarctica than you'd think!'

**Review by DarklingMuse — COMMENTED** [review]
One suggestion added

File: 02_profiles/08_i-don-t-want-everyone-to-know-where-i-live-what-do-i-do.md (hunk 1 above)
Comment (sylee-) [reply 02_profiles/08_i-don-t-want-everyone-to-know-where-i-live-what-do-i-do.md]: Done

**Review by DarklingMuse — APPROVED** [review]
Lookin' good!

**Comment by DarklingMuse** [comment]
This has changed location / been updated since we started. New PR is here: https://github.com/fetlife/help/pull/110

**Pull request closed without merging**